synapse/lib/lmdblayer.py

```diff
@@ -508,17 +508,34 @@ def _storPropSetCommon(self, buid, penc, bpkey, pvpref, univ, valu, indx):
             oldv, oldi = s_msgpack.un(byts)
             if oldi is not None:
 
-                self.layrslab.delete(pvpref + oldi, pvvalu, db=self.byprop)
+                if isinstance(oldi, bytes):
```

Comment:
we have a bunch of copy/past duplication here.  We could just wrap an instance of ``bytes`` into a tuple and avoid some of the copy / paste code in this file
```
if isinstance(oldi, bytes):
    oldi = (oldi,)
for oldibyts in oldi:
    .... do stuff ...
```

Reply:
Agreed.  For now, we'll let performance win over code efficiency and assume it will be cleaned up in the eventual layer format re-jigger.